playwright.config.ts

```diff
@@ -32,9 +32,14 @@ export default defineConfig({
 
   /* Configure projects for major browsers */
   projects: [
+    { name: "setup", testMatch: /auth.setup\.ts/ },
```

Comment:
Always ensures we get an authenticated browser state before running any other tests